mkosi/sandbox.py

```diff
@@ -392,6 +392,11 @@ def is_relative_to(one: str, two: str) -> bool:
     return os.path.commonpath((one, two)) == two
 
 
+def makedirs(path: str, mode: int = 511, exist_ok: bool = True) -> None:
```

Comment:
Slightly better readability
```suggestion
def makedirs(path: str, mode: int = 0o777, exist_ok: bool = True) -> None:
```